src/transformers/models/ibert/utils/quant_modules.py

```diff
@@ -0,0 +1,538 @@
+import torch
+import time
+import math
+import numpy as np
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.multiprocessing as mp
+from torch.nn import Linear as _linear
+from torch.nn import Embedding as _Embedding
+from torch.nn import Module, Parameter
+from .quant_utils import *
+
+from ....utils import logging
+logger = logging.get_logger(__name__)
+
+class QuantEmbedding(Module):
+    """
+    Class to quantize given Embedding layer
+
+    Parameters:
+    weight_bit : int
```

Comment:
could we try to have the docstring more in the format of https://github.com/huggingface/transformers/blob/570218878a502469f2ce62a33bc62be9b3f89b4a/src/transformers/modeling_utils.py#L1251?